db/redis.js

```diff
@@ -53,6 +53,16 @@ async function addDoorTestSensorData(request, response){
     response.status(200).json("OK")
 }
 
+const addIM21DoorSensorData = (locationid, doorSignal) => {
+    if(doorSignal=="closed"||doorSignal=="open"){
```

Comment:
You probably want to use `DOOR_STATE` from `SessionStateDoorEnum.js` here instead of `open` and `closed`.